repo/meta/index.json

```diff
@@ -31,6 +31,18 @@
       "versions":{
         "alpha":"0"
       }
+    },
+    {
+      "currentVersion":"1.3.0",
+      "description":"Spark is a fast and general cluster computing system for Big Data",
+      "name":"spark",
+      "tags":[
+        "mesosphere",
+        "framework"
```

Comment:
I'm not even sure what the tags are for, I don't want to start introducing random tags though. Do we have standard on what tags should we be using?